app/src/main/java/at/roteskreuz/stopcorona/screens/base/CoronaBaseActivity.kt

```diff
@@ -70,18 +70,6 @@ open class CoronaBaseActivity(@LayoutRes layout: Int = R.layout.framelayout) : B
                 debugViewModel.addOutgoingMessageYellow()
                 true
             }
-            R.id.debugMenuInfectionMessagesIncomingRed -> {
```

Comment:
please also have a look at `R.menu.debug` and remove obsolete entries

Reply:
updated